features/aanschrijfwijze.feature

```diff
@@ -66,12 +66,12 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
         - de adellijke titel en het predikaat worden met kleine letters opgenomen
         - de adellijke titel of het predikaat van de partner wordt niet opgenomen bij aanduiding naamgebruik "E" (eigen naam)
         - de adellijke titel of het predikaat van de persoon wordt niet opgenomen bij aanduiding naamgebruik "P" (partner naam)
-        - de adellijke titel van de partner wordt niet opgenomen als geslachtsaanduiding van de persoon ongelijk is aan "V" (vrouw) of als de geslachtsaanduiding van de partner ongelijk is aan "M" (man)
+        - de adellijke titel van de partner wordt niet opgenomen als geslachtsaanduiding van de persoon gelijk is aan "O" (onbekend)
         - het predikaat begint met een hoofdletter als de aanschrijfwijze begint met een predikaat 
         - het predikaat van de partner wordt niet opgenomen
         - het predikaat van de persoon wordt opgenomen achter het streepje en voor eigen voorvoegsel en geslachtsnaam bij aanduiding naamgebruik "V" (partner naam voor eigen naam)
-        - bij meerdere actuele (niet ontbonden) huwelijken/partnerschappen, wordt de naam gegevens van de eerste partner (oudste relatie) gebruikt voor het samenstellen van de aanschrijfwijze
-        - bij geen actuele huwelijk/partnerschap en meerdere ontbonden huwelijken/partnerschappen, wordt de naam gegevens van de laatst ontbonden partner gebruikt voor het samenstellen van de aanschrijfwijze
+        - bij meerdere actuele (niet ontbonden) huwelijken/partnerschappen, wordt de naam gegevens van de eerste partner (oudste relatie) gebruikt voor het samenstellen van de aanschrijfwijze.naam
```

Comment:
"wordt de naam gegevens" --> "worden de naamgegevens"

Reply:
ook hier vraag ik me af of het nodig is .naam toe te voegen. deze regel geldt ook voor aanschrijfwijze.aanspreekvorm

In de regel staat m.i. al duidelijk dat het gaat om de naam in aanschrijfwijze. Dan hoeft dat wat mij betreft hier niet meer te worden herhaald (vooral omdat het voor aanspreekvorm eigenlijk ook geldt)

Reply:
Aangepast